src/content/docs/99-manual/nodejs/issues.md

```
@@ -0,0 +1,136 @@
+```bash
+# 删除 node_modules 和 lock 文件重装
+rm -rf node_modules package-lock.json
+npm install
+
+# 清除 npm 缓存
+npm cache clean --force
+```
+
+### peerDependencies 警告
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)